test/get_previous_releases.py

```diff
@@ -148,6 +148,27 @@ def download_binary(tag, args) -> int:
     ret = subprocess.run(['tar', '-zxf', tarball, '-C', tag,
                           '--strip-components=1',
                           'bitcoin-{tag}'.format(tag=tag[1:])]).returncode
+
+    if tag >= "v23" and platform == "arm64-apple-darwin":
+        # Starting with v23.0 there are arm64 binaries for ARM (M1/M2) macs, but they have to be signed to run
```

Comment:
nit: Mentioning of exact models, i.e. M1/M2, does not look future proof. Maybe pointing at arm64 architecture is enough?

Reply:
Until I started getting into Docker I hadn't been all that aware of cpu architecture details. I wouldn't have immediately associated `arm64` with an M1 mac.

But I don't feel strongly either way about the comment.